mllib/src/main/scala/org/apache/spark/ml/feature/OneHotEncoder.scala

```diff
@@ -56,17 +56,26 @@ class OneHotEncoder(override val uid: String) extends Transformer
     new BooleanParam(this, "dropLast", "whether to drop the last category")
   setDefault(dropLast -> true)
 
+  /**
+   * Param for the output attr prefix. If not specified, a prefix will be automatically generated.
+   * @group param
+   */
+  final val outputAttrPrefix: Param[String] =
```

Comment:
We try to keep ML attributes handling under the hood. So this parameter might be surprising to many users. Maybe we don't need to keep prefix and `_is_` in the generated feature names. Instead of "country_is_US`, we can have just `US` as the feature name, but under group `country` or whatever the output column names is.